src/test/java/seedu/address/logic/commands/ListBuyersCommandTest.java

```diff
@@ -48,4 +48,34 @@ public void testExecuteBuyers() throws CommandException {
     public void keywordEqualsBuyers() {
         assertEquals("buyers", ListBuyersCommand.KEY_WORD);
     }
+
+    @Test
+    void equals_sameCommand_returnsTrue() {
+        // Arrange
+        ListBuyersCommand listBuyersCommand1 = new ListBuyersCommand();
+        ListBuyersCommand listBuyersCommand2 = new ListBuyersCommand(); // Same command
+
+        // Act & Assert
```

Comment:
Same as the previous comment, I feel that these comments don't need to be added :)